_maps/RandomRooms/storagepod.dmm

```diff
@@ -0,0 +1,459 @@
+/obj/item/device/instrument/bikehorn,
+/obj/item/device/instrument/piano_synth,
+/obj/item/device/instrument/saxophone,
+/turf/open/floor/plating,
+/area/greytopia)
+"r" = (
+/obj/structure/table/reinforced,
+/obj/effect/spawner/lootdrop/three_course_meal,
+/obj/item/candle/infinite{
+	pixel_x = 9;
+	pixel_y = -3
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"s" = (
+/obj/structure/rack,
+/obj/item/device/flashlight/flare/torch{
+	pixel_x = -4;
+	pixel_x = -2;
+	pixel_y = 7
+	},
+/obj/item/device/flashlight/flare/torch{
+	pixel_x = 3;
+	pixel_y = 5
+	},
+/obj/item/device/flashlight/flare/torch{
+	pixel_x = 3;
+	pixel_y = 5
+	},
+/obj/item/device/flashlight/flare/torch{
+	pixel_x = -3;
+	pixel_x = 9;
+	pixel_y = -3
+	},
+/obj/item/device/flashlight/lantern,
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"t" = (
+/obj/structure/closet/radiation,
+/obj/machinery/light/small{
+	dir = 8
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"u" = (
+/obj/structure/rack,
+/obj/item/device/instrument/glockenspiel,
+/obj/item/device/instrument/harmonica,
+/obj/item/device/instrument/trombone,
+/turf/open/floor/plating,
+/area/greytopia)
+"v" = (
+/obj/structure/rack,
+/obj/item/storage/belt/utility/full/engi{
+	pixel_y = -5
+	},
+/obj/item/storage/belt/utility/full/engi{
+	pixel_y = -5
+	},
+/obj/item/storage/belt/utility/full/engi{
+	pixel_x = -3;
+	pixel_x = -2;
+	pixel_y = 3
+	},
+/obj/item/kitchen/knife/combat/survival{
+	pixel_y = 7
+	},
+/obj/item/kitchen/knife/combat/survival{
+	pixel_x = 3;
+	pixel_y = -1
+	},
+/turf/open/floor/plating{
+	icon_state = "platingdmg2"
+	},
+/area/greytopia)
+"w" = (
+/obj/machinery/sleeper{
+	dir = 8
+	},
+/obj/machinery/light/small{
+	dir = 4
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"x" = (
+/obj/structure/rack,
+/obj/item/storage/belt/mining{
+	pixel_x = -1;
+	pixel_y = -4
+	},
+/obj/item/storage/belt/mining{
+	pixel_x = -3;
+	pixel_x = -3;
+	pixel_y = 4
+	},
+/obj/item/tank/jetpack/void{
+	pixel_x = 6;
+	pixel_y = 2
+	},
+/obj/item/tank/jetpack/void{
+	pixel_x = 6;
+	pixel_y = 2
+	},
+/obj/item/tank/jetpack/void,
+/turf/open/floor/plating{
+	icon_state = "panelscorched"
+	},
+/area/greytopia)
+"y" = (
+/obj/structure/rack,
+/obj/item/stack/sheet/cloth/ten{
+	pixel_x = -2;
+	pixel_x = -2;
+	pixel_y = 6
+	},
+/obj/item/stack/sheet/glass/fifty,
+/obj/item/stack/sheet/metal/fifty{
+	pixel_x = -1;
+	pixel_x = 5;
+	pixel_y = -6
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"z" = (
+/obj/machinery/suit_storage_unit/standard_unit,
+/turf/open/floor/plating,
+/area/greytopia)
+"A" = (
+/obj/machinery/sleeper{
+	dir = 8
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"B" = (
+/obj/structure/fans/tiny,
+/obj/machinery/door/airlock/hatch,
+/turf/open/floor/plating,
+/area/greytopia)
+"C" = (
+/obj/machinery/light/small{
+	dir = 4
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"D" = (
+/turf/closed/wall,
+/area/greytopia)
+"E" = (
+/obj/structure/falsewall{
+	layer = 3.4
```

Comment:
layer?

Reply:
Theres a poster on this turf and without this var the false wall eats the poster